### PR TITLE
create new line in marshmallow and a new md file called ferrets 

### PR DESCRIPTION
**Pull request by dcfwight**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/marshmallow.md [hunk 1]
```diff
@@ -606,4 +606,6 @@ Just then you recall the news story of what is causing the
 
 You escape into a [recording studio](more_cowbell/more_cowbell.md)
 
-And then you stumble upon something [completely different](http://www.atlasobscura.com/)
\ No newline at end of file
+And then you stumble upon something [completely different](http://www.atlasobscura.com/)
+
+A snarling ferret king who fixes you with a [steely glare](../ferrets/Ferret.md)
```
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/ferrets/Ferret.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The evil ferret king laughed as he fed you to his hordes of squealling ferret babies.
```
Comment (casey-collab) [inline english/ferrets/Ferret.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/marshmallow.md]: The `../` at the beginning is unnecessary, since the `ferrets` directory is directly inside of the `english` directory.

**Comment by cbuckey-uda** [comment]
Thanks for making the update, and I'm glad you enjoyed the course! @casey-collab, please take another look

File: english/ferrets/Ferret.md (hunk 2 above)
Comment (casey-collab) [inline english/ferrets/Ferret.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/marshmallow.md (hunk 1 above)
Comment (casey-collab) [inline english/marshmallow.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/ferrets/Ferret.md (hunk 2 above)
Comment (casey-collab) [inline english/ferrets/Ferret.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
It looks like the file `english/ferrets/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/marshmallow.md (hunk 1 above)
Comment (cbuckey-uda) [inline english/marshmallow.md]: Hm, I'm not sure what went wrong, but it looks like your commit did not actually update this link.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**